docs/dev/new_check_howto.md

```diff
@@ -360,7 +352,7 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
 
 ##### Example manifest config
 
-Our example integration has a very simple manifest, the bulk of which is generated by the tooling. Note that the `guid` must be unique (and valid), so do *not* use the one from this example.
+Our example integration has a very simple `awesome/manifest.json`, the bulk of which is generated by the tooling. Note that the `guid` must be unique (and valid), so do *not* use the one from this example.
```

Comment:
Do we want to mention here that the guid is auto generated for you via the create command and the default value should be fine? (https://github.com/DataDog/integrations-core/blob/a8f1422378c367b9ef1908deb4d1e554217734db/datadog_checks_dev/datadog_checks/dev/tooling/create.py#L78)

Reply:
A reasonable suggestion for a (near-)future PR. 😄